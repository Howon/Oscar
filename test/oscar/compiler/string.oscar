def main() => unit = {
  string s2 = "string2";
  Println("string1");
  Println(s2);

  string escp = "\"\'\\\\n\ttest";
  Println(escp);
}
