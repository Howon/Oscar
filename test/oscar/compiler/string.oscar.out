string1
string2
"'\\n	test
